build.gradle

```diff
@@ -8,10 +8,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.7.20'
+    ext {
+        kotlin_version = '1.7.20'
+        customCSSFile = projectDir.toString() + "/dokka/fr-backstage-styles.css"
+        customLogoFile = projectDir.toString() + "/dokka/logo-icon.svg"
+        customTemplatesFolder = file(projectDir.toString() + "/dokka/templates")
+    }
     repositories {
         google()
-        jcenter()
```

Comment:
Jcenter repositorties are deprecated , so using mavencentral is fine. But @spetrov  keep an eye this when you release ....